### ISSUE TITLE
implement MouseWheel ftns (scrolling)

### ISSUE THREAD

**Issue by bburdette** [obv-mikhail/InputBot#11]
Hey!  First off, thanks for the awesome lib.  Today I made a program to let me use my phone as a mouse - it works!  mousepage on crates.io.

Now I'm looking to get scrolling to work, but looks like the relevant functions are unimplemented!  MouseWheel.scroll_ver and MouseWheel.scroll_hor.

**Comment by obv-mikhail** [comment]
@bburdette thank you!

I am guessing that you are using Linux? X11 or Wayland?

I will look into how to implement the scrolling.

**Comment by bburdette** [comment]
Yep, linux.  I'm not sure if its X11 or wayland, truthfully.  Its nixos with xmonad as the window manager.
